### DISCUSSION TITLE
The car in flight tries to align horizontally

### DISCUSSION THREAD

**Discussion by NikolayKolomiytsev** [JustInvoke/Randomation-Vehicle-Physics#56] [Q&A]
If my car is going at some angle (drifting) and the front wheels lose traction at that moment (bump, springboard, whatever) the car immediately tries to align horizontally, I could not find the reason for this, but it looks terribly unrealistic, especially  when sometimes it even "rolls" in the opposite direction during this alignment, according to my tests, I can say for sure that this is not "VehicleAssist" and "FlipControl", if you disable 2 of these scripts, the situation will not change, increasing angularDrag does not help (in reasonable values  without freezing the car at all).  Perhaps you know what it could be?

**Comment by JustInvoke** [comment]
I'm having a difficult time reproducing it consistently but I might have been able to a couple times. Could you share a video of the problem so I know what to look for? If the problem is what I think it is then perhaps the friction and/or suspension forces are over-correcting, but I can't really fix it unless I can manage to reproduce it consistently.
